user/serializers.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from django.contrib.auth.password_validation import password_changed
```

Comment:
The `password_changed` function is imported but not used in the code. If it is not needed, consider removing this import to clean up the code.